tests/lay-fxl-xhtml-icb_error/EPUB/page_001.xhtml

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>Invalid viewport setting yields device dimensions</title>
+    <meta name="viewport" content="anything, width=1000, height=1000" />
```

Comment:
This brings up an interesting question - if we can successfully parse out width and height from this (which we do), should we use it?  I believe the intent of the test is that it should default to device width/height.  Anyhow, a discussion to be had somewhere else.  :-)

Reply:
Right, this is an issue (and the test reflects the current spec), and it is being discussed in https://github.com/w3c/epub-specs/issues/2442 about now... We would love to have your reactions, @davemanhall